seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/TaskController.java

```diff
@@ -71,6 +72,12 @@ Result<InstanceSimpleInfoRes> tmpExecute(@RequestBody @NotNull ExecuteReq req) {
         return Result.success(iTaskService.tmpExecute(req));
     }
 
+    @GetMapping("/queryInstanceLog")
+    @ApiOperation(value = "query instance log", httpMethod = "GET")
+    Result<InstanceLogRes> queryInstanceLog(@RequestParam long taskInstanceId) {
+        return Result.success(iTaskService.queryInstanceLog(taskInstanceId));
+    }
+
     @PostMapping("/kill")
     @ApiOperation(value = "kill running instance", httpMethod = "POST")
     Result<Void> kill(@RequestParam Long instanceId) {
```

Comment:
hi, the front end seems to lose precision when passing the long data type

Reply:
Don't worry, it won't happen.

Reply:
Use `@JSONField(serializeUsing = ToStringSerializer.class)` can avoid precision loss when the back-end passes Long to the front-end.
And when front-end uses string data type to call back-end's api ,  if backend use Long to get the param, it's still work.